src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -1369,61 +1366,56 @@ static bool isValidNullableConversion(
-            {
-                return false;
-            }
-
-            var parameters = method.Parameters;
-
-            // https://github.com/dotnet/csharplang/blob/main/proposals/csharp-11.0/low-level-struct-improvements.md#scoped-mismatch
-            // The compiler will report a diagnostic for _unsafe scoped mismatches_ across overrides, interface implementations, and delegate conversions when:
-            // - The method returns a `ref struct` or returns a `ref` or `ref readonly`, or the method has a `ref` or `out` parameter of `ref struct` type, and
-            // ...
-            int nRefParametersRequired;
-            if (method.ReturnType.IsRefLikeOrAllowsRefLikeType() ||
-                (method.RefKind is RefKind.Ref or RefKind.RefReadOnly))
-            {
-                nRefParametersRequired = 1;
-            }
-            else if (parameters.Any(p => (p.RefKind is RefKind.Ref or RefKind.Out) && p.Type.IsRefLikeOrAllowsRefLikeType()))
-            {
-                nRefParametersRequired = 2; // including the parameter found above
-            }
-            else
-            {
-                return false;
-            }
-
-            // ...
-            // - The method has at least one additional `ref`, `in`, `ref readonly`, or `out` parameter, or a parameter of `ref struct` type.
-            int nRefParameters = parameters.Count(p => p.RefKind is RefKind.Ref or RefKind.In or RefKind.RefReadOnlyParameter or RefKind.Out);
-            if (nRefParameters >= nRefParametersRequired)
-            {
-                return true;
-            }
-            else if (parameters.Any(p => p.RefKind == RefKind.None && p.Type.IsRefLikeOrAllowsRefLikeType()))
-            {
-                return true;
-            }
-
-            return false;
-        }
-
         /// <summary>
         /// Returns true if a scoped mismatch should be reported as an error rather than a warning.
         /// </summary>
         internal static bool ReportInvalidScopedOverrideAsError(MethodSymbol baseMethod, MethodSymbol overrideMethod)
         {
             // https://github.com/dotnet/csharplang/blob/main/proposals/csharp-11.0/low-level-struct-improvements.md#scoped-mismatch
             // The diagnostic is reported as an error if the mismatched signatures are both using C#11 ref safety rules; otherwise, the diagnostic is a warning.
-            return baseMethod.UseUpdatedEscapeRules && overrideMethod.UseUpdatedEscapeRules;
+            return baseMethod.UseUpdatedEscapeRules && overrideMethod.UseUpdatedEscapeRules &&
+                // We have removed exceptions to the scoped mismatch error reporting, but to avoid breaks
+                // we report the new scenarios (previously exempted) as warnings in C# 12 and earlier.
+                // https://github.com/dotnet/roslyn/issues/76100
+                (overrideMethod.DeclaringCompilation.LanguageVersion > LanguageVersion.CSharp12 || usedToBeReported(baseMethod));
```

Comment:
Downgraded the new errors to warnings in C# 12 and lower to lessen the break per offline discussion with Jared. (We think breaking in C# 13 is fine since it's new.)